### DISCUSSION TITLE
Recover account with 2FA enabled

### DISCUSSION THREAD

**Discussion by aelnosu** [community/community#72891] [Code Security]
### Select Topic Area

Product Feedback

### Body
FYI: I do NOT need to recover my account. This is just a suggestion.

In https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#requesting-help-with-two-factor-authentication

step 5:To complete your recovery request, you'll need to verify an alternative authentication factor. Choose a recovery verification factor.
I see there is a method of using ssh key but it will be even better to allow PGP key attached with the Github account and S/Mime key with email that is registered on the Github account which is trusted by a CA.

**Comment by 4lexxe** [comment]
F

**Reply by aelnosu** [reply]
G
